faunadb-java/src/test/java/com/faunadb/client/ClientSpec.java

```diff
@@ -2086,6 +2086,33 @@ public void shouldTestIdentity() throws Exception {
     }
   }
 
+  @Test
+  public void shouldTestCreateAccessProvider() throws Exception {
+    String roleName = randomStartingWith("role_");
+    String providerName = randomStartingWith("provider_");
+    String issuerName = randomStartingWith("issuer_");
+    String fullUri = randomStartingWith("https://") + ".auth0.com";
+
+    adminClient.query(CreateRole(Obj(
+      "name", Value(roleName),
+      "privileges", Obj(
+        "resource", Databases(),
+        "actions", Obj("read", Value(true))
+      )
+    ))).get();
+
+    Value provider = adminClient.query(
+      CreateAccessProvider(Obj(
+      "name", Value(providerName),
+      "issuer" , Value(issuerName),
+      "jwks_uri" , Value(fullUri),
+      "membership", Arr(Role(roleName))))
```

Comment:
It seems the `membership` field needs to be updated to `roles`. @marrony, could you confirm that's the case?

Reply:
It is indeed the case :+1: 
I have performed the renaming for both the [Java](https://github.com/fauna/faunadb-jvm/pull/267/commits/d8bc638cef207d58a759bb2b46a46fefac81a17f) & [Scala PR](https://github.com/fauna/faunadb-jvm/pull/265/commits/05e26a1a36e5248f42b91bbd115de018001939da).